### PR TITLE
upload FRSKD and GKD Loss

### PR DESCRIPTION
**Pull request by WangChen0902**
FRSKD和CD中的GKD Loss
已知风险项：只测试了BiFPNc而未测试BiFPN，未测试DistillationGTCELoss改名对其他模块的影响

### CODE REVIEW

File: ppcls/arch/distill/frskd_bifpn.py [hunk 1]
```diff
@@ -0,0 +1,406 @@
+import math
```
Comment (littletomatodonkey) [inline ppcls/arch/distill/frskd_bifpn.py:1]: +license

File: ppcls/arch/distill/frskd_bifpn.py [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+import paddle
+import paddle.nn as nn
+import paddle.nn.functional as F
+
+__all__ = ['BiFPNc', 'BiFPN']
```
Comment (littletomatodonkey) [inline ppcls/arch/distill/frskd_bifpn.py:6]: 去掉BiFPN

File: ppcls/arch/distill/frskd_bifpn.py [hunk 3]
```diff
@@ -0,0 +1,406 @@
...
+
+
+class BiFPNc(nn.Layer):
+    def __init__(self,
```
Comment (littletomatodonkey) [inline ppcls/arch/distill/frskd_bifpn.py:10]: 参考文献

File: ppcls/arch/distill/frskd_bifpn.py [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+                 network_channel=[64, 128, 256, 512],
+                 num_classes=1000,
+                 repeat=1,
+                 depth=2,
+                 width=2,
+                 num_features=4,
+                 preact=False,
+                 keys=["blocks[1]", "blocks[3]", "blocks[5]", "blocks[7]"]):
+        super(BiFPNc, self).__init__()
```
Comment (littletomatodonkey) [inline ppcls/arch/distill/frskd_bifpn.py:19]: `super().__init__()`

File: ppcls/loss/atloss.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+
+import paddle.nn as nn
+import paddle.nn.functional as F
+
+
+class ATLoss(nn.Layer):
+    def __init__(self, p=2):
```
Comment (littletomatodonkey) [inline ppcls/loss/atloss.py:20]: +reference

File: ppcls/configs/ImageNet/Distillation/resnet18_distill_frskd.yaml [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+Loss:
+  Train:
+    - DistillationGTCELoss:
+        weight: 1.0
+        model_names: ["Student"]
+        key: logits
+    - DistillationGTCELoss:
+        weight: 1.0
+        model_names: ["Student"]
+        key: logits_fpn
```
Comment (littletomatodonkey) [inline ppcls/configs/ImageNet/Distillation/resnet18_distill_frskd.yaml:54]: for more detailed inforamation of FRSKD output, you can refer to xxx
